elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/controllers/Utils.java

```diff
@@ -8,11 +8,25 @@
 
 import static com.yahoo.elide.core.dictionary.EntityDictionary.NO_VERSION;
 
+import java.util.HashMap;
 import java.util.Map;
 
+import javax.ws.rs.core.HttpHeaders;
+
 public class Utils {
     public static String getApiVersion(Map<String, String> requestHeaders) {
         return requestHeaders.getOrDefault("ApiVersion",
                 requestHeaders.getOrDefault("apiversion", NO_VERSION));  //For tomcat
     }
+
+    public static Map<String, String> removeAuthHeaders(Map<String, String> requestHeaders) {
```

Comment:
We can commonize this logic with HeaderUtils.

Spring has a class called HttpHeaders and JaXRS has a class called MultivalueMap.  Both extends `Map<String, List<String>> `.

We can commonize removing auth headers and the like by using the Java standard.  

This means requestScope should probably have a Map<String, List<String>> instead of a Multivalued map.